index.html

```diff
@@ -5,8 +5,45 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="icon" type="image/x-icon" href="/img/street-food.ico">
     <title>Document</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
 </head>
-<body >
-    
+<body class="wrapper">
```

Comment:
Body is a type element that appears once, let's use less generic name for it's selector